proto/vtctldata.proto

```diff
@@ -205,6 +206,11 @@ message Shard {
   topodata.Shard shard = 3;
 }
 
+message WorkflowOptions {
+  string tenant_id = 1;
+  string source_keyspace_alias = 2;
+}
+
 // TODO: comment the hell out of this.
```

Comment:
Hey, who wrote this TODO???

Reply:
https://github.com/planetscale/vitess/commit/d0fdccaa4bb3ddc102bd88e11db1e58d904410c3